specification/databricks/resource-manager/Microsoft.Databricks/preview/2023-09-15-preview/databricks.json

```diff
@@ -0,0 +1,2025 @@
+      "enum": [
+        "Accepted",
+        "Running",
+        "Ready",
+        "Creating",
+        "Created",
+        "Deleting",
+        "Deleted",
+        "Canceled",
+        "Failed",
+        "Succeeded",
+        "Updating"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "PeeringProvisioningState": {
+      "type": "string",
+      "readOnly": true,
+      "description": "The current provisioning state.",
+      "enum": [
+        "Succeeded",
+        "Updating",
+        "Deleting",
+        "Failed"
+      ],
+      "x-ms-enum": {
+        "name": "PeeringProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "ManagedIdentityConfiguration": {
+      "type": "object",
+      "description": "The Managed Identity details for storage account.",
+      "properties": {
+        "principalId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The objectId of the Managed Identity that is linked to the Managed Storage account.",
+          "readOnly": true
+        },
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The tenant Id where the Managed Identity is created.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of Identity created. It can be either SystemAssigned or UserAssigned.",
+          "readOnly": true
+        }
+      }
+    },
+    "EnhancedSecurityComplianceDefinition": {
+      "type": "object",
+      "description": "Status of settings related to the Enhanced Security and Compliance Add-On.",
+      "properties": {
+        "automaticClusterUpdate": {
+          "type": "object",
+          "description": "Feature that allows for automatic restart of compute resources",
+          "$ref": "#/definitions/AutomaticClusterUpdateDefinition"
+        },
+        "complianceSecurityProfile": {
+          "type": "object",
+          "description": "Feature that helps customers meet the applicable security requirements of some compliance standards.",
+          "$ref": "#/definitions/ComplianceSecurityProfileDefinition"
+        },
+        "enhancedSecurityMonitoring": {
+          "type": "object",
+          "description": "Feature that provides an enhanced disk image and additional security monitoring agents",
+          "$ref": "#/definitions/EnhancedSecurityMonitoringDefinition"
+        }
+      }
+    },
+    "AutomaticClusterUpdateDefinition": {
+      "type": "object",
+      "description": "Feature that allows for automatic restart of compute resources",
+      "properties": {
+        "maintenanceWindow": {
+          "type": "object",
+          "description": "Schedule for automatic restart of compute resources",
+          "$ref": "#/definitions/MaintenanceWindowDefinition"
+        },
+        "value": {
```

Comment:
>value [](http://example.com/codeflow?start=9&length=5)

Something like "activation", or maybe "status" seems like a more descriptive name for this property.